spec/util/inputSanitizer-spec.js

```diff
@@ -107,6 +107,8 @@ describe('Input Santizer for Protected sheet', function () {
       ring: '',
       quadrant: '',
       isNew: '',
+      hasMovedIn: '',
```

Comment:
Thinking about this more, I like the idea of having one field that can be utilized instead of `isNew` since a blip can be new OR moved in OR moved out OR a refresh.

Reply:
I agree, a new field `status` or similar  

Would you be willing to make this a breaking change, or want to support backwards compatibility with datasets that are using `isNew`? 

Can you elaborate on your definition of `refresh`.  Is this for blips that are removed from the radar and re-appear, or something different?

Reply:
I'd prefer making it backward compatible. So, users should have the option to _either_ use `isNew` or `status`. If there's a value in `status`, we would use that field.

And here's where the differences between TW Tech Radar and BYOR come back 😊 The "refresh" I'm referring to is what shows as "No change" on https://www.thoughtworks.com/radar. Basically, the blip has appeared before and we're blipping it again, but it stays in the same ring as last time. In BYOR, this would be a `false` for `isNew`.

Reply:
Ok great, that clears things up.  Would you want to keep it as `refresh`, or prefer something different?

Reply:
Since `refresh` isn't too intuitive, I'm open to suggestions! I was thinking maybe `existing` but not sure on that. `notNew` seems not great too.

Reply:
I think either `existing` or `noChange` would work. Whichever we use should match the terminology used in the legend.